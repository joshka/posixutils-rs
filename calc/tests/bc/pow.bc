scale = 2
a = 2.000 ^ 10
scale(a)
a

scale = 4
a = 2.000 ^ 10
scale(a)
a

scale = 4
a = 2.000 ^ 1
scale(a)
a

scale = 0
a = 2 ^ -1
scale(a)
a

a = 2.00 ^ -1
scale(a)
a

scale = 2
a = 2.00 ^ -1
scale(a)
a

scale = 10
2 ^ -1
2 ^ -2
89.290 ^ -1

scale = 0
2 ^ 1
2 ^ 2
3.489 ^ 4
10 ^ 10
0 ^ 0
0 ^ 38
1 ^ 1
1 ^ 30
quit
