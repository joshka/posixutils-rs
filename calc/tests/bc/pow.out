3
1024.000
4
1024.0000
3
2.000
0
0
0
0
2
0.50
0.5000000000
0.2500000000
0.0111994624
2
4
148.184
10000000000
1
0
1
1
